pkg/proxy/ipvs/proxier.go

```diff
@@ -607,34 +606,29 @@ func NewLinuxKernelHandler() *LinuxKernelHandler {
 // GetModules returns all installed kernel modules.
 func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 	// Check whether IPVS required kernel modules are built-in
+	var nfconntrack bool
 	kernelVersionStr, err := handle.GetKernelVersion()
 	if err != nil {
 		return nil, err
 	}
-	kernelVersion, err := version.ParseGeneric(kernelVersionStr)
-	if err != nil {
-		return nil, fmt.Errorf("error parsing kernel version %q: %v", kernelVersionStr, err)
-	}
-	ipvsModules := utilipvs.GetRequiredIPVSModules(kernelVersion)
 
-	var bmods, lmods []string
+	ipvsModules := utilipvs.GetRequiredIPVSModules()
+
+	var bmods []string
 
 	// Find out loaded kernel modules. If this is a full static kernel it will try to verify if the module is compiled using /boot/config-KERNELVERSION
 	modulesFile, err := os.Open("/proc/modules")
```

Comment:
I would split this part of the code as a helper function (private) as:

(pardon typos/code style)
```
// Go docs here
function isStaticKernel() (bool, error) {
    if modulesFile, err := os.Open("/proc/modules");  err == os.ErrNotExist {
                       return true, nil
   } 
   
  if err != nil {
                return nil, fmt.Errorf("Failed to read file /proc/modules with error %v", err)
  }

  return false, nil 
}
```